frontend/src/uxbox/util/geom/point.cljs

```diff
@@ -9,13 +9,17 @@
 
 (ns uxbox.util.geom.point
   (:refer-clojure :exclude [divide])
-  (:require [uxbox.util.math :as mth]
-            [cognitect.transit :as t]))
+  (:require
+   [cuerdas.core :as str]
+   [uxbox.util.math :as mth]
+   [cognitect.transit :as t]))
 
 ;; --- Point Impl
 
 (defrecord Point [x y])
 
+(defn s [{:keys [x y]}] (str "(" x "," y ")"))
```

Comment:
Maybe we can implement the toString method to point, and use the point as is... (minor, don't change right now, we can experiment with this later).